gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -59,29 +60,24 @@ def print_cuking_command(
         "cuKING on the files created by --prepare-cuking-inputs."
     )
     # TOTAL_SHARDS calculates the total number of shards using the formula k(k+1)/2.
-    cuking_command = (
-        f"""\
-        SPLIT_FACTOR = {cuking_split_factor} \\"""
-        + """
+    cuking_command = f"""\
+        SPLIT_FACTOR={cuking_split_factor} \\
         TOTAL_SHARDS=$((SPLIT_FACTOR * (SPLIT_FACTOR + 1) / 2)) \\
-        for SHARD_INDEX in $(seq 0 $((TOTAL_SHARDS - 1))); do \\
-            cuKING_dsub \\"""
-        + f"""
-            --location={location} \\
-            --machine-type={machine_type} \\
-            --accelerator-count={accelerator_count} \\
-            --accelerator-type={accelerator_type} \\
-            --command="cuking \\
-            --input_uri="{cuking_input_path}" \\
-            --output_uri="{cuking_output_path}" \\
-            --requester_pays_project={requester_pays_project} \\
-            --kin_threshold={min_emission_kinship} \\"""
-        + """
-            --split-factor=${SPLIT_FACTOR} \\
-            --shard-index=${SHARD_INDEX}" \\
+        for SHARD_INDEX in $(seq 0 $((TOTAL_SHARDS - 1))); do
+        cuKING_dsub \\
+        --location={location} \\
+        --machine-type={machine_type} \\
+        --accelerator-count={accelerator_count} \\
+        --accelerator-type={accelerator_type} \\
+        --command="cuking \\
+        --input_uri="{cuking_input_path}" \\
+        --output_uri=f"{cuking_output_path}/out_split_${{SHARD_INDEX}}.parquet" \\
+        --requester_pays_project={requester_pays_project} \\
```

Comment:
FMK, Is there a reason `input_uri` needs quotes but `requester_pays_project` does not?  It looks like this is  exactly what ran in the NB, well what was pasted, so I'm good approving this but I think in hindsight, I'd make the nested  `--command` into a defined variable above the cuking_command defintion to make it a bit more readable (the opposite of this run-on sentence). This is sorta what co-pilot is getting at but I dont think that needs to be done now since we ran this already.

Reply:
i think it was just that requester_pays_project uses more simple characters/no variables although in this case i don't think it matters